### PR TITLE
test

### PR DESCRIPTION
**Pull request by mvallotto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
   "description": "Easy-to-use error reporting & error code generation",
   "main": "index.js",
   "scripts": {
+    "test": "./node_modules/.bin/nyc ./node_modules/.bin/mocha test/index.js && ./node_modules/.bin/nyc report --reporter=clover"
     "test": "./node_modules/.bin/nyc ./node_modules/.bin/mocha test/* --sort && ./node_modules/.bin/nyc report --reporter=clover"
```
Comment (finwo) [inline package.json]: Why?
It was already there

**Review by finwo — CHANGES_REQUESTED** [review]

File: index.js [hunk 2]
```diff
@@ -31,16 +29,29 @@ var tter = module.exports = function( options ) {
    * @returns {*}
    */
   function reporter( level, description ) {
-    if ( 'undefined' === typeof description ) { description = level ; level = options.defaultLevel; }
-    if(isNaN(level)) level = tter.level[level.toUpperCase()] || options.defaultLevel;
-    if ( level > options.level ) return description;
+    if ( 'undefined' === typeof description ) { 
+      description = level; 
+      level = options.defaultLevel;
```
Comment (finwo) [inline index.js]: No `=` alignment

File: index.js [hunk 3]
```diff
@@ -31,16 +29,29 @@ var tter = module.exports = function( options ) {
...
+    if ( level > options.level ) {
+      return description;
+    }
     var code = options.scopeHash + '.' + slash(description),
-        err  = { code: code, level: tter.level[level], description: description };
-
+        err  = { 
+          code: code, 
+          level: tter.level[level], 
+          description: description
```
Comment (finwo) [inline index.js]: No `:` alignment

**Review by finwo — CHANGES_REQUESTED** [review]

File: index.js (hunk 2 above)
Comment (mvallotto) [reply index.js]: done

File: index.js (hunk 3 above)
Comment (mvallotto) [reply index.js]: done

**Review by finwo — APPROVED** [review]

**Pull request merged**